source/jormungandr/jormungandr/interfaces/common.py

```diff
@@ -54,3 +54,17 @@ def handle_poi_infos(add_poi_info_param, bss_stands_param):
         add_poi_info_param.append("bss_stands")
 
     return any(value in add_poi_info_param for value in ['bss_stands', 'car_park'])
+
+
+def split_uri(uri):
+    """
+    Split uri into an output list
+    :param uri: concatenated uri params
+    :return: the splitted uri list
+    """
+    uris = []
```

Comment:
```python
if not uri: # try without this line
    return [] # try without this line
return uri.strip('/').split('/')
```

Reply:
sweet

Reply:
what about the part `uri=uri[:-1]`?

Reply:
apparently, `strip('/'`) erase the first and the last `/` of the string.
before 
`/tit/toto/tata/`
it remains afer
`titi/toto/tata`

Reply:
you know @xl, he is a touchy pythonic guy